lib/pages/incident.dart

```diff
@@ -200,4 +215,41 @@ class _IncidentPageState extends State<IncidentPage> {
       }).toList(),
     );
   }
+
+  Widget showRemoveDialog(BuildContext context) {
+    // set up the buttons
+    Widget cancelButton = FlatButton(
+      child: Text("Cancel"),
+      onPressed: () {
+        Navigator.pop(context);
+      },
+    );
+    Widget deleteButton = FlatButton(
+      child: Text(
+        "Delete",
+        style: TextStyle(color: Colors.red),
+      ),
+      onPressed: () {
+        _delete();
+        Navigator.of(context).popUntil(ModalRoute.withName("/home"));
+      },
+    );
+    // set up the AlertDialog
+    AlertDialog alert = AlertDialog(
+      title: Text("Are you sure"),
```

Comment:
Just missed the question mark
```suggestion
      title: Text("Are you sure?"),
```

Reply:
Done!!